Package.swift

```diff
@@ -8,16 +8,16 @@ let package = Package(
     ],
     dependencies: [
         // Swift Promises, Futures, and Streams.
-        .package(url: "https://github.com/vapor/async.git", "1.0.0-beta.1"..<"1.0.0-beta.2"),
+        .package(url: "https://github.com/vapor/async.git", from: "1.0.0-rc"),
```

Comment:
I've had issues with this syntax with the pre-release identifiers of it not working and/or takes aaages to resolve. Might be better to stick to the syntax of `"1.0.0-rc.1"..<"1.0.0-rc.2"` - also means that any changes won't be pulled in until deliberately done so.

Opening up for discussion! 😁

Reply:
![](http://s2.quickmeme.com/img/eb/eb5c40ea92b5b583d67faa509bf8cd37ec3f28c27ec76c88dce201a087a267e4.jpg)

I can change it to the syntax you suggest, basically the only reason I opted for the `from:` parameter is because I saw it in other packages ([Core](https://github.com/vapor/core/blob/master/Package.swift), [Engine](https://github.com/vapor/engine/blob/master/Package.swift) and possibly others) and just wanted to follow the suit 😄